src/wormhole/Governance.sol

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: Apache 2
+pragma solidity >=0.6.12 <0.9.0;
+
+import "wormhole-solidity-sdk/interfaces/IWormhole.sol";
+
+contract Governance {
+    IWormhole immutable wormhole;
+
+    error InvalidGovernanceChainId(uint16 chainId);
+    error InvalidGovernanceContract(bytes32 contractAddress);
+    error GovernanceActionAlreadyConsumed(bytes32 digest);
+
+    error NotRecipientChain(uint16 chainId);
+    error NotRecipientContract(address contractAddress);
+
+    bytes32 constant CONSUMED_GOVERNANCE_ACTIONS_SLOT =
```

Comment:
Why is this slot hash calculated differently from every other slot?

Reply:
that was a bit of an oversight, fixed here: https://github.com/wormhole-foundation/example-native-token-transfers/compare/48c246a92d11421e1d545711815739f8d61f02e6..2c7096ebe08e650b3b57394d6fa9d3706795a116

the reason it doesn't mention `ntt` is because it's technically independent of `ntt` and this exact same contract could be used to govern other contracts too, since it's fully generic.

Reply:
Nice, change looks good. Yeah happy that it's not `ntt`, I agree it's independent